definitions/variable/technology/power-plant.yaml

```diff
@@ -165,6 +165,11 @@
     description: Efficiency of pumping for a pumped-storage hydro unit
     unit: '%'
     skip-region-aggregation: true
+- Inflows|Electricity|Hydro|Pumped Storage:
+    description: Inflows into a reservoir expressed in energy
+    unit:
+    - MWh
```

Comment:
Is it necessary to allow both units here? I'd prefer to only allow one, to reduce the risk of errors.

Reply:
I can keep the `GW` and `GWh` in the respective variables.
I got your point. Thus, I will take off the extra unit and keep only one.